06-building-a-program-start-to-finish/writing-tests/Output/invertedIndexTest.js

```diff
@@ -0,0 +1,79 @@
+/* Test Setup */
+const myInvertedIndex = new InvertedIndexClass();
+const book = require('./../books.json');
+const emptyBook = require('./../bookempty.json');
+const wrongBook = require('./../bookwrongformat.json');
+const notBook = require('./../notBook.json');
+const books = require('./../books3.json');
```

Comment:
A base filename should exactly match the name of its default export. [See link](https://github.com/airbnb/javascript#naming--filename-matches-export)